packages/fxa-settings/src/components/Settings/PageTwoStepAuthentication/index.test.tsx

```diff
@@ -149,6 +150,15 @@ describe('step 1', () => {
     expect(screen.getByTestId('manual-code')).toBeInTheDocument();
     expect(GleanMetrics.accountPref.twoStepAuthScanCodeLink).toHaveBeenCalled();
   });
+
+  it('sends metrics when step 1 is submitted without viewing manual code', async () => {
+    await act(async () => {
+      render();
+    });
```

Comment:
Not sure why so many of these tests have `await act`, we generally try to avoid this. Works file with a simple render!

```suggestion
      render();
```

Reply:
I thought that I had tried this without await and they didn't pass, but I'll try again locally.

Reply:
Ah yeah, this was the error that made me add it:
```
  console.error
    Warning: You called act(async () => ...) without await. This could lead to unexpected testing behaviour, interleaving multiple act calls and mixing their scopes. You should - await act(async () => ...);
```

Reply:
I'm going to land it as is, if that's alright.